tests/test_command.py

```diff
@@ -2,10 +2,20 @@
 from nose.tools import assert_equals
 
 from remotespark.livyclientlib.command import Command
+from remotespark.livyclientlib.livysession import LivySession
 from remotespark.utils.constants import SESSION_KIND_SPARK
 import remotespark.utils.configuration as conf
 from . import test_livysession as tls
 
+def _create_session(kind=SESSION_KIND_SPARK, session_id=-1,
+                    sql_created=False, http_client=None, spark_events=None):
+    if http_client is None:
+        http_client = MagicMock()
+    if spark_events is None:
+        spark_events = MagicMock()
+    ipython_display = MagicMock()
+    session = LivySession(http_client, {"kind": kind}, ipython_display, session_id, sql_created, spark_events)
+    return session
```

Comment:
I admit that pasting the old method here isn't the cleanest way. But before my change, it called a member method in another test class, which already looks odd. Ideally we should extract the common method into a base test suite/a test helper class.